Vagrantfile

```diff
@@ -0,0 +1,40 @@
+  Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision :shell, privileged: false, inline: <<-SHELL
+    sudo apt-get update
+
+    # TODO: Install pyenv prerequisites
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    # TODO: Install pyenv
+    sudo rm -rf ~/.pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.profile
+  
+    source ~/.profile
+
+    pyenv install 3.7.9
+    pyenv global 3.7.9
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+  SHELL
+
+    config.trigger.after :up do |trigger|
+  trigger.name = "Launching App"
+  trigger.info = "Running the TODO app setup script"
+  trigger.run_remote = {privileged: false, inline: "
+  # Install dependencies and launch
+  poetry install
```

Comment:
I expect this fails because it can't find your .toml file, right?

When this script runs, it will run from the home directory of the user (who is called "vagrant"): `/home/vagrant`. But your project files are in the `/vagrant` directory